lib/masdb/initializer.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Masdb.Initializer do
+  def start_link do
+    {:ok, spawn_link(__MODULE__, :initialize_node, [])}
+  end
+
+  def initialize_node do
+    nodes = Masdb.Node.list()
+    {:ok, answers} = Masdb.Node.DistantSupervisor.query_remote_nodes_until(
```

Comment:
Would it be a good idea to have a "maximum_attempt" parameter, so we are able to get an error if the node is querying ad vitam æternam?